oracle_server/oracle_server.go

```diff
@@ -391,7 +392,7 @@ func (os *OracleServer) reportWithCommitment(newRound uint64, lastRoundData *typ
 		return err
 	}
 
-	os.logger.Info("oracle client left fund", "balance", balance.String())
+	os.logger.Info("oracle server account left fund", "address", os.key.Address, "balance", balance.String())
```

Comment:
"left fund" can be removed from the log. Or say remaining balance. there is a balance field getting printed sperately.

Reply:
> "left fund" can be removed from the log. Or say remaining balance. there is a balance field getting printed sperately.

Resolved in bd07764ac76b51a6cd8ae44a169d0b74eb116d63